mix.exs

```diff
@@ -56,9 +56,7 @@ defmodule RealtimeSigns.Mixfile do
       {:uuid, "~> 1.1", only: :test},
       {:quantum, "~> 3.0"},
       {:phoenix, "~> 1.7.0"},
-      {:phoenix_html, "~> 3.0"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
```

Comment:
Don't need this one anymore either, I think?

Reply:
There's still a reference to it, but I guess we could strip that out too.